simple-social-icons.php

```diff
@@ -199,6 +206,13 @@ function __construct() {
 			),
 		) );
 
+		/**
+		 * If setting this filter to true, we recommend mentioning in your theme documentation
+		 * that Simple Social Icons widget settings will noy display styling options, as your
```

Comment:
Another: noy → not

Reply:
What can I say about that? 🤦‍♂️ version 2!

Reply:
Resolved by the last commit.